package.json

```
@@ -19,7 +19,7 @@
   },
   "packageManager": "pnpm@7.11.0",
   "peerDependencies": {
-    "effector": "^22.8.3"
+    "effector": "^23.0.0"
```

```suggestion
    "effector": "^22.8.8 | ^23.0.0"
```

---

but
```suggestion
    "effector": "^22.8.8 || ^23.0.0"
```
https://semver.npmjs.com

---

@victordidenko  thx